delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -0,0 +1,370 @@
+import six
+
+from oslo_log import log as logging
+from delfin import exception
+from delfin.common import constants, alert_util
+from delfin.drivers.utils.ssh_client import SSHPool
+from delfin.drivers.utils.tools import Tools
+
+try:
+    import xml.etree.cElementTree as ET
```

Comment:
What is the difference between line10 and 12?

Reply:
The correct thing isexcept ImportError:
    import xml.etree.ElementTree as ET

Reply:
The correct thing is   except ImportError:
    import xml.etree.ElementTree as ET      I made a mistake in writing. I wrote an extra C